src/components/AttachmentModal.tsx

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Was this blank line why the linter was failing? Maybe just try `npm run prettier` from `main` as an alternative.